CHANGELOG.txt

```diff
@@ -2,6 +2,14 @@
 MABWiser CHANGELOG
 =====================
 
+-------------------------------------------------------------------------------
+August, 01, 2021 1.12.0
+-------------------------------------------------------------------------------
+major:
+- New Algorithm: TreeBandit as a decision-tree based neighborhood policy.
+- Many thanks to @irmakbky for her contribution!
```

Comment:
Thank you heaps Irmak!! This is a great win for MABWiser!

Reply:
Thank you!